index.js

```diff
@@ -16,6 +30,18 @@ module.exports = {
     this.projectChecker = new VersionChecker(this.project);
     this.emberVersion = this.projectChecker.for('ember-source').version;
 
+    if (this.emberVersion === null) {
+      let bowerrcPath = path.join(this.root, '.bowerrc');
```

Comment:
Needs to be project.root I think (I copied my original snippet from where `this` was the Project base class)